db/knex.js

```diff
@@ -1,3 +1,7 @@
+var debug = require('debug')('app:config')
+
 var env = process.env.NODE_ENV || 'development';
 var config = require('../knexfile.js')[env];
+debug(config);
 module.exports = require('knex')(config);
```

Comment:
The `debug` package is a debug logger rather than an interactive debugger. If you want that, you'll want to use `node-inspector`. That being said, I pretty much never use `node-inspector` as `debug` is generally all I need.